project/src/test/java/com/google/sps/cloudstoragetests/GcsFilePathTest.java

```diff
@@ -0,0 +1,76 @@
+package com.google.sps;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Date;
+import java.util.ArrayList;
+import java.util.HashMap;
+import com.google.appengine.api.blobstore.FileInfo;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.data.CloudStorageLibrary;
+
+@RunWith(JUnit4.class)
+public final class GcsFilePathTest {
+    //Empty Map of files
+    private static final Map<String, List<FileInfo>> NO_FILES = Collections.emptyMap();
+
+    //Defining sample files for testing
+    private static final Date date = new Date();
+    private static final Long size = 27262L;
```

Comment:
Is this specific size relevant? If so, it might be nice to have a comment explaining why you chose this size.

However, if it's just a random number for testing purposes, then I don't think you need to explain it.

Reply:
Initially I had it as the typical size of a file, but I was struggling on how to initialize a long, and I finally found out that it needs an L at the end. But through all the changes the number did not stay the same. I can change it though.